moto/ec2/models.py

```diff
@@ -2877,6 +2881,66 @@ def associate_vpc_cidr_block(
         vpc = self.get_vpc(vpc_id)
         return vpc.associate_vpc_cidr_block(cidr_block, amazon_provided_ipv6_cidr_block)
 
+    def create_vpc_endpoint(self,
+                             vpc_id,
+                             service_name,
+                             type=None,
+                             policy_document=False,
+                             route_table_ids=None,
+                             subnet_ids=[],
+                             network_interface_ids=[],
+                             dns_entries=None,
+                             client_token=None,
+                             security_group=None,
+                             tag_specifications=None,
+                             private_dns_enabled=None
+                            ):
+
+        vpc_endpoint_id = generate_vpc_end_point_id(vpc_id)
+
+        #validates if vpc is present or not.
+        self.get_vpc(vpc_id)
+
+        if type == "interface" or "Interface ":
```

Comment:
This is evaluated as ```(type == "interface") or ("Interface")```
The string `Interface` will always evaluate to True, regardless of the outcome of `type == interface` - so this statement doesn't do what you think it does
```suggestion
        if type and type.lower() == "interface":
```